drivers/soundwire/qcom.c

```diff
@@ -877,8 +868,7 @@ static int qcom_swrm_remove(struct platform_device *pdev)
 	struct qcom_swrm_ctrl *ctrl = dev_get_drvdata(&pdev->dev);
 	int err;
 
-	sdw_delete_bus_master(&ctrl->bus);
-	err = sdw_master_device_del(ctrl->md);
+	err = sdw_master_device_del(&ctrl->md);
```

Comment:
I think something is missing here: I don't see where the type of `ctrl->md` changed from a pointer to an embedded structure?

Reply:
yes, also flagged by kbuild, I need to change this for qcom.c